src/Framework/AspNetCoreAnalyzers/src/Analyzers/DiagnosticDescriptors.cs

```diff
@@ -248,4 +248,13 @@ internal static class DiagnosticDescriptors
         DiagnosticSeverity.Info,
         isEnabledByDefault: true,
         helpLinkUri: AnalyzersLink);
+
+    internal static readonly DiagnosticDescriptor UseCreateHostBuilderInsteadOfCreateWebHostBuilder = new(
+        "ASP0029",
```

Comment:
ASP0029 is the diagnostic code we are using for the Experimental attribute on the validation sutff so this will have to be ASP0030.